llmfoundry/data/finetuning/dataloader.py

```diff
@@ -552,15 +552,6 @@ def _build_collate_fn(
                                     1)],
                                              skip_special_tokens=False,
                                              clean_up_tokenization_spaces=True))
-                        print(
-                            '\033[92m{}\033[00m\n'.format('CONTEXT:  '),
-                            tokenizer.decode(batch['input_ids'][
-                                j,
-                                torch.logical_and(
-                                    is_subseq, batch['bidirectional_mask'][j] ==
-                                    1)],
-                                             skip_special_tokens=False,
-                                             clean_up_tokenization_spaces=True))
```

Comment:
You shouldn't delete these lines. Instead, find a different way to slice out the context. `batch['bidirectional_mask'][j] == 1` was just a convenient way to do that. The context occurs where `attention_mask==1` and `labels==_HF_IGNORE_INDEX`.